src/app/shared/custom-components/map/map.component.ts

```diff
@@ -56,43 +52,29 @@ export class MapComponent implements OnInit, OnDestroy {
 
   onChangeOffice(office: Office): void {
     this.offerRequest$.unsubscribe();
-    const markers = this.initOfficeMarkers(office);
-    this.applyMarkers(markers);
+    this.markers = this.initOfficeMarkers(office);
   }
 
   onChangeOffers(): void {
     this.offerRequest$ = this.offerService
       .getOffers({ city: this.city })
       .subscribe((offers) => {
-        const markers = this.initOffersMarkers(offers);
-        this.applyMarkers(markers);
+        console.log(offers);
```

Comment:
do you need this console.log? or did you forget?

Reply:
forgot